frontend/src/components/data-table/charts/charts.tsx

```diff
@@ -243,6 +246,12 @@ export const ChartPanel: React.FC<{
 
   const [selectedChartType, setSelectedChartType] =
     useState<ChartType>(chartType);
+  const [formCollapsed, setFormCollapsed] = useState(false);
+
+  const chartContainerRef = useRef<HTMLDivElement>(null);
+  useResizeObserver({
+    ref: chartContainerRef as React.RefObject<Element>,
+  });
```

Comment:
Would this work to remove the type cast?

```suggestion
  const { ref: chartContainerRef } = useResizeObserver();
```

Reply:
great suggestion, thanks!